tests.out

```diff
@@ -0,0 +1,2 @@
+python "E:\git3\coreclr\tests\runtest.py" -arch x64 -build_type Checked -test_env E:\Test\testenv.cmd --generate_layout
```

Comment:
Looks like a personal log file you don't want to add.

Reply:
Oops! Thanks for noticing